aviary/subsystems/propulsion/motor/model/motor_premission.py

```diff
@@ -0,0 +1,81 @@
+import openmdao.api as om
+import numpy as np
+
+from aviary.variable_info.variables import Aircraft, Dynamic
+from aviary.utils.aviary_values import AviaryValues
+from aviary.subsystems.propulsion.motor.model.motor_map import MotorMap
+
+
+class MotorPreMission(om.Group):
+    """
+    Calculate electric motor mass for a single motor
+    """
+
+    def initialize(self):
+        self.options.declare(
+            "aviary_inputs", types=AviaryValues,
+            desc="collection of Aircraft/Mission specific options",
+            default=None,
+        )
+        self.name = 'motor_premission'
+
+    def setup(self):
+
+        # Determine max torque of scaled motor
+
+        # We create a set of default inputs for this group so that in pre-mission,
+        # the group can be instantiated with only scale_factor as an input.
+        # without inputs and it will return the max torque
+        # based on the non-dimensional scale factor chosen by the optimizer.
+        # The max torque is then used in pre-mission to determine weight of the system.
+        self.set_input_defaults(Dynamic.Mission.THROTTLE, 1.0, units=None)
+
+        # TBD I'm worried that the above code won't set these in pre-mission correctly
```

Comment:
This comment should be removed. But we should also check that it is correctly setting throttle to 1 in pre-mission first.